### PR TITLE
Updated OKHTTP Timeouts  ⏱️ 

### PR DESCRIPTION
**Pull request by jumaallan**
### **User description**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

___

### **PR Type**
Enhancement

___

### **Description**
- Increased OKHTTP timeouts to match Android SDK

- Updated version from 2.1.6-SNAPSHOT to 2.1.7-SNAPSHOT

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>SmileIdentityService.java</strong><dd><code>Increased OKHTTP timeouts for better network reliability</code>&nbsp; </dd></summary>
<hr>

src/main/java/smile/identity/core/SmileIdentityService.java

<li>Increased call timeout from 30 to 120 seconds<br> <li> Added explicit connect timeout of 60 seconds<br> <li> Added explicit read timeout of 60 seconds<br> <li> Added explicit write timeout of 60 seconds

</details>

  </td>
  <td><a href="https://github.com/smileidentity/smile-identity-core-java/pull/55/files#diff-f47343a68db6caea3ddbd6ec13c8618312c8af1b1a0677a0ec5c4ab879050c26">+6/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>VERSION</strong><dd><code>Bumped version number</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

VERSION

- Updated version from 2.1.6-SNAPSHOT to 2.1.7-SNAPSHOT

</details>

  </td>
  <td><a href="https://github.com/smileidentity/smile-identity-core-java/pull/55/files#diff-7b60b8e351cbb80c47459ffe2c79f1a26404871f49294780fe47ad0e58c09350">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> <details> <summary>  Need help?</summary><li>Type <code>/help how to ...</code> in the comments thread for any questions about PR-Agent usage.</li><li>Check out the <a href="https://qodo-merge-docs.qodo.ai/usage-guide/">documentation</a> for more information.</li></details>

### CODE REVIEW

**Review by beastawakens — APPROVED** [review]

File: src/main/java/smile/identity/core/SmileIdentityService.java [hunk 1]
```diff
@@ -25,7 +25,12 @@ public class SmileIdentityService {
     static Logger logger = LogManager.getLogger(SmileIdentityService.class);
 
     public SmileIdentityService(String server) {
-        this(server, new OkHttpClient.Builder().callTimeout(30, java.util.concurrent.TimeUnit.SECONDS));
+        this(server, new OkHttpClient.Builder()
+                .callTimeout(120, java.util.concurrent.TimeUnit.SECONDS)
+                .connectTimeout(60, java.util.concurrent.TimeUnit.SECONDS)
+                .readTimeout(60, java.util.concurrent.TimeUnit.SECONDS)
+                .writeTimeout(60, java.util.concurrent.TimeUnit.SECONDS)
+        );
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply src/main/java/smile/identity/core/SmileIdentityService.java:33]: Good suggestion - we can create this as an issue and work on it later
